x-pack/metricbeat/module/aws/_meta/config.yml

```diff
@@ -16,7 +16,14 @@
   secret_access_key: '${AWS_SECRET_ACCESS_KEY:""}'
   session_token: '${AWS_SESSION_TOKEN:""}'
   default_region: '${AWS_REGION:us-west-1}'
-  cloudwatch_metrics:
+  metrics:
+    - namespace: AWS/EC2
+      name: ["CPUUtilization", "DiskWriteOps"]
+      tags.resource_type_filter: ec2:intance
+      dimensions:
+        - name: InstanceId
+          value: i-0686946e22cf9494a
```

Comment:
I think this is a good example, but won't work by default for everyone, as the instanceID is specific to your scenario. It's probably better to keep it, but commented out as an example

Reply:
Good point! I will comment these three lines out. Thanks!